inclusive-design/wecount.inclusivedesign.ca
# feat: add the social media info for WeCount linkedIn (resolves #276)

## Description

Add the social media info for WeCount LinkedIn.

## Steps to test

1. Scroll to the bottom of a page;
2. The linked in icon should be displayed in the "social media" section;
3. Click the link brings the page to the WeCount project on LinkedIn;
4. Focus/hover styles for the linked in icon should be in sync with other social media icons;
5. Select other UIO themes;
6. The color of the icon should change accordingly as well as the focus/hover styles.

**Expected behavior:** 

See above.
